lib/src/base-styles/_nx-card.scss

```diff
@@ -0,0 +1,175 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-card-container {
+  margin: $nx-spacing-md 0;
+}
+
+.nx-card {
+  background-color: #fff;
+  border-radius: $nx-border-radius;
+  box-sizing: border-box;
+  padding: $nx-spacing-l $nx-spacing-xxl;
+}
+
+.nx-card__call-out {
+  .nx-icon {
+    font-size: 48px;
+  }
+
+  .nx-icon--large {
+    font-size: 64px;
+  }
+}
+
+.nx-card__call-out-text {
+  @include bold();
+
+  font-size: 32px;
+}
+
```

Comment:
I'm thinking `.nx-card-container--column` should include `container-vertical`.  Right now the first and last card's margins collapse out of the parent.  The parent has the same margins already, so it doesn't actually do anything, but just for consistency and cleanliness...

Reply:
> I'm thinking .nx-card-container--column should include container-vertical. Right now the first and last card's margins collapse out of the parent. The parent has the same margins already, so it doesn't actually do anything, but just for consistency and cleanliness...

Oh it already does have it.  Well it's not working, I think you'll have to make some one-off more specific selectors.

Reply:
> > I'm thinking .nx-card-container--column should include container-vertical. Right now the first and last card's margins collapse out of the parent. The parent has the same margins already, so it doesn't actually do anything, but just for consistency and cleanliness...
> 
> Oh it already does have it. Well it's not working, I think you'll have to make some one-off more specific selectors.

The tweak is only required on the row layout due to the bottom margin on the cards themselves: https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/d2501f44b33e904af1521ebcc0eb33663e963af2